src/Uno.UI/UI/Xaml/Controls/Repeater/_Tracing.cs

```diff
@@ -0,0 +1,24 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Microsoft.UI.Xaml.Controls
+{
+	internal static class _Tracing
+	{
+		[Conditional("TRACE")]
+		public static void REPEATER_TRACE_INFO(string text, params object[] parameters)
+			=> Console.WriteLine(text, parameters);
```

Comment:
Codacy found an issue: [Remove this logging statement.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=5863680)